go.mod

```diff
@@ -1,10 +1,12 @@
 module github.com/enterprise-contract/ec-policies
 
-go 1.21
+go 1.21.2
+
+toolchain go1.21.3
 
 require (
+	github.com/enterprise-contract/ec-cli v0.0.0-20231107103908-d7e137f0b777
```

Comment:
dependabot won't keep these up to date. (That's probably ok, just calling it out)

Reply:
Seems that Renovate [will](https://github.com/renovatebot/renovate/blob/f9d1cade0804a36f40bd0afb327fe4aee4c69023/lib/modules/datasource/go/index.spec.ts#L163)